mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -660,6 +651,25 @@ func resourceDataprocCluster() *schema.Resource {
 											},
 										},
 									},
+									"confidential_instance_config": {
```

Comment:
Any reason that we only add `ForceNew` to the subfield `enable_confidential_compute` but not the parent field `confidential_instance_config`? Can removing and adding `enable_confidential_compute` be supported without needing to recreate the resource?

Reply:
As of the `resourceDataprocClusterUpdate` call has not handled the change in `cluster_config.0.gce_cluster_config` so to update a value of `enable_confidential_compute` we need to support the `ForceNew`.  otherwise we need to write the `update` functionality for whole object of `gce_cluster_config`.